backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -65,10 +68,21 @@ export const InitEmailTransactionalMiddleware = (
   type ReqBodyWithId = ReqBody & { emailMessageTransactionalId: string }
 
   function convertMessageModelToResponse(
-    message: EmailMessageTransactional,
+    message: EmailMessageTransactionalWithRelations,
     excludeParams = false,
     ccEmails?: { cc?: string[]; bcc?: string[] }
```

Comment:
Why do we still need this? Can u update the other usage of this function to just use the `message` params to pass along the cc emails?